helpers/measures.ts

```diff
@@ -56,6 +56,23 @@ export function isDiffMetric(metricKey: string): boolean {
   return metricKey.indexOf('new_') === 0;
 }
 
+/*
+ * Conditional decimal count for QualityGate-impacting measures
+ * (e.g. Coverage %)
+ */
+export function getDecimalsNumber(threshold: number | undefined, value: number) {
```

Comment:
I would swap the parameters since the `threshold` can be undefined.

Reply:
Make sense if threshold is optional, but I actually think it is not. Not sure why this could be undefined though.